tests/foreman/cli/test_template.py

```diff
@@ -8,7 +8,7 @@
 
 :CaseComponent: ProvisioningTemplates
 
-:Assignee: ogajduse
+:Assignee: sganar
```

Comment:
It appears that /tests/foreman/cli/test_provisioning.py module is in need of updating for the same, right?

Reply:
Updated